### PR TITLE
adding phpunit tests for 9 modules.

### PR DESCRIPTION
**Pull request by tedstrauss**
adding automated tests for the following modules: user_accounts, timepoint_flag, survey_accounts, reliability, next_stage, mri_violations, mri_upload, instrument_manager, instrument_list, instrument_builder.

### CODE REVIEW

File: modules/instrument_builder/test/instrument_builderTest.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ * @package  Loris
+ * @author   Ted Strauss <ted.strauss@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+
+require_once __DIR__ . "/../../../test/integrationtests/LorisIntegrationTest.class.inc";
+class instrumentBuilderTestIntegrationTest extends LorisIntegrationTest
+{
+    protected $url = 'http://localhost/main.php?test_name=instrument_builder';
```
Comment (driusan) [inline modules/instrument_builder/test/instrument_builderTest.php]: This hides the variable from the LorisIntegrationTest class which looked up the URL from the config settings.

Can you remove this class variable and instead call get as:

$this->webDriver->get($this->url . "?test_name=instrument_builder");

Comment (tedstrauss) [inline modules/instrument_builder/test/instrument_builderTest.php]: sure

Comment (driusan) [inline modules/instrument_builder/test/instrument_builderTest.php]: I just realized that the pull request which does that isn't merged yet. You'll have to also rebase  onto #973

File: modules/mri_violations/test/mri_violationsTest.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+{
+    protected $url = 'http://localhost/main.php?test_name=mri_violations';
+
+    /**
+     * Tests that, when loading the Mri_violations module, some
+     * text appears in the body.
+     *
+     * @return void
+     */
+    function testMriViolationsDoespageLoad()
```
Comment (driusan) [inline modules/mri_violations/test/mri_violationsTest.php]: Can you add a test to make sure the 2 submenus load:

?test_name=mri_violations&submenu=mri_protocol_violations

and

?test_name=mri_violations&submenu=mri_protocol_check_violations

File: modules/survey_accounts/test/survey_accountsTest.php [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+{
+    protected $url = 'http://localhost/main.php?test_name=survey_accounts';
+
+    /**
+     * Tests that, when loading the Survey accounts module, some
+     * text appears in the body.
+     *
+     * @return void
+     */
+    function testSurveyAccountsDoespageLoad()
```
Comment (driusan) [inline modules/survey_accounts/test/survey_accountsTest.php]: Can you also add a test for the subpage loading:

?test_name=survey_accounts&subtest=add_survey

File: modules/user_accounts/test/user_accountsTest.php [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+{
+    protected $url = 'http://localhost/main.php?test_name=user_accounts';
+
+    /**
+     * Tests that, when loading the User accounts module, some
+     * text appears in the body.
+     *
+     * @return void
+     */
+    function testUserAccountsDoespageLoad()
```
Comment (driusan) [inline modules/user_accounts/test/user_accountsTest.php]: There are 2 submodules to check that they load: edit_user, my_preferences subpages

File: modules/survey_accounts/test/survey_accountsTest.php (hunk 3 above)
Comment (tedstrauss) [inline modules/survey_accounts/test/survey_accountsTest.php]: ok

File: modules/mri_violations/test/mri_violationsTest.php (hunk 2 above)
Comment (tedstrauss) [inline modules/mri_violations/test/mri_violationsTest.php]: ok

File: modules/instrument_builder/test/instrument_builderTest.php (hunk 1 above)
Comment (tedstrauss) [inline modules/instrument_builder/test/instrument_builderTest.php]: i see... ok i'll do that

**Comment by driusan** [comment]
This needs to update the travis.yml file too so that the tests are run.

**Pull request merged**